docs/overview.md

```diff
@@ -1,6 +1,6 @@
 # API Elements Overview
 
-**Version**: 1.0.0-rc1
+**Version**: 1.1.0
```

Comment:
Sounds great. I'll let you all decide what version you'd like to use for this. Bump the rc number?